sdk-tests/components/actorstatestore.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: mysql-actorstatestore
+spec:
+  type: state.mysql
+  version: v1
+  metadata:
+    - name: connectionString
+      value: "root:test@tcp(127.0.0.1:3306)/?allowNativePasswords=true"
+    - name: cleanupIntervalInSeconds
+      value: "1"
+    - name: actorStateStore
+      value: "true"
+scopes:
+  - actorstateit-statefulactorservice
```

Comment:
Needs to add the other actor apps too. This way, other tests don't require actor state store.